### PR TITLE
fix(ui): display junit

### LINKED ISSUE #4693: fix(ui): tests failed aren't displayed
when the pipeline ends the tests tab is not up-to-date:

Screenshot 2019-10-22 at 18 03 32

### LINKED ISSUE #4711: fix(ui): relaunch btn, resync with big payload
if payload is too big, the resynchronize button can't be accessed from ui.

### PR DESCRIPTION
**Pull request by yesnault**
close #4693
close #4711

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-ui#11235.0 ✘  
* Compile
  * Build UI ✔ 
  * Lint Translate ✘  
  * TU All -   
  * TU model ✔ 
  * TU shared ✔ 
  * TU store ✔ 
  * TU views ✔

File: ui/src/app/views/workflow/run/node/test/table/test.table.component.ts [hunk 1]
```diff
@@ -1,88 +1,136 @@
...
+            lineWrapping: false,
+            autoRefresh: true,
+            readOnly: true
+        };
+
+        this.filter = f => {
+            const lowerFilter = f.toLowerCase();
+            return d => {
+                if (lowerFilter.indexOf('status:') === 0) {
+                    return lowerFilter.toLowerCase().indexOf(d.status) >= 7;
```
Comment (bnjjj) [inline ui/src/app/views/workflow/run/node/test/table/test.table.component.ts]:
```suggestion
                    return lowerFilter.indexOf(d.status) >= 7;
```

File: ui/src/app/views/workflow/run/node/test/table/test.table.component.ts [hunk 2]
```diff
@@ -1,88 +1,136 @@
...
+    getFailureString(fs: Array<Failure>): string {
+        if (!fs) {
+            return '';
+        }
+        let r = '';
+        for (const f of fs) {
+            if (f.message && f.message !== '') {
+                r += f.message + '<hr>';
+            }
+            if (f.value && f.value !== '') {
```
Comment (bnjjj) [inline ui/src/app/views/workflow/run/node/test/table/test.table.component.ts]:
```suggestion
            if (f.value) {
```

File: ui/src/app/views/workflow/run/node/test/table/test.table.component.ts [hunk 3]
```diff
@@ -1,88 +1,136 @@
...
+        this.testCaseSelected = tc;
     }
+
+    getFailureString(fs: Array<Failure>): string {
+        if (!fs) {
+            return '';
+        }
+        let r = '';
+        for (const f of fs) {
+            if (f.message && f.message !== '') {
```
Comment (bnjjj) [inline ui/src/app/views/workflow/run/node/test/table/test.table.component.ts]:
```suggestion
            if (f.message) {
```

File: ui/src/app/views/workflow/run/node/node.html [hunk 4]
```diff
@@ -43,12 +43,30 @@
             </a>
         </sm-menu>
     </div>
-    <div class="scrollingContent" ngx-auto-scroll>
+    <!-- we want to autoscroll page only for building pipeline -->
+    <div *ngIf="selectedTab === 'pipeline' && nodeRun.status === 'Building'; else notBuilding;" class="scrollingContent" ngx-auto-scroll>
```
Comment (bnjjj) [inline ui/src/app/views/workflow/run/node/node.html]: Can you use PipelineStatus from `pipeline.model.ts` for Building status please

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11237.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Pull request merged**